test/test_models.py

```diff
@@ -390,7 +403,10 @@ def _check_input_backprop(model, inputs):
     "swin_v2_b",
 ]
 for m in slow_models:
-    _model_params[m] = {"input_shape": (1, 3, 64, 64)}
+    if m not in _model_params:
+        _model_params[m] = {"input_shape": (1, 3, 64, 64)}
+    else:
+        _model_params[m]["input_shape"] = (1, 3, 64, 64)
```

Comment:
Should we just make model_params a defaultdict?

Reply:
@NicolasHug could you explain why we should use defaultdict here?

Reply:
It would avoid having to check for existence of the `m` key:

```py
_model_params = {
    "resnet50": {"input_shape": (1, 3, 64, 64)},
    #... 
}
_model_params = defaultdict(dict, _model_params)

_model_params["SomeNewKeyThatDoesntExist"]["input_shape"] = (1, 2, 3, 4)
_model_params["resnet50"]["input_shape"] = (1, 2, 3, 4)
print(_model_params)
# defaultdict(<class 'dict'>, {'resnet50': {'input_shape': (1, 2, 3, 4)}, 'SomeNewKeyThatDoesntExist': {'input_shape': (1, 2, 3, 4)}})
```

Reply:
I think another (possibly better?) way to handle that is something like

```py
for m in slow_models:
    _model_params[m] = dict(_model_params.get(m, dict()), **{"input_shape": (1, 3, 64, 64)})
```

(not sure about the exact syntax but you get the idea)

Reply:
got it, I think your second way is better (less line), will use that one. Thanks!

Reply:
> less line

It looks like more lines as I was illustrating its usage, but it should actually result in less characters :)

(eitherway both are fine!)